### DISCUSSION TITLE
Connect to Garmin Watch - Such as Forerunner 955/965

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [doudar/SmartSpin2k#483] [Q&A] [answered]
I just found out that some Garmin watches can control smart trainers...when I try searching for the SS2K on the watch it doesn't find it.  Has anyone connected to a Garmin watch as a smart trainer (not just to get power/cadence)?

I'm wondering if it's an issue with the SS2K or if I have to unpair from my current setup (even though my current setup is powered off and it's not connected).

**Comment by doudar** [comment, marked as answer]
My guess is that the watch supports ant+ only and not BLE.

**Reply by Lunchtime0614** [reply]
Ahh - crap...good call  - https://support.garmin.com/en-US/?faq=CHnxYj6F2OA6tN4QRRjEJ8&identifier=886725&tab=topics

**Reply by Lunchtime0614** [reply]
@doudar  - has there been any thought in supporting ANT+ FE-C?  I'm assuming the hardware is not capable?

**Reply by Lunchtime0614** [reply]
Disregard - I see a post from Feb where someone asked something similar and that ANT+ is not on the roadmap.

**Comment by doudar** [comment]
The hardware isn’t currently capable. We considered adding it on rev 3, but it would have increased the board cost/complexity by quite a bit as well as a lot of software dev work.
